### DISCUSSION TITLE
going nuts.

### DISCUSSION THREAD

**Discussion by Kiekendeko** [hlky/stable-diffusion#245] [Q&A]
i could write it in issues, but that would mean that the actual issue is related to the software comparatively to my own local incompetence with finishing the installation of the program.

I'm also sorry for using this as a help guide, but i cannot find a better place to go to especially as it links here.

I've been using this guide [here](https://rentry.org/GUItard) to try and hopefully make the damned thing work, including re-downloading the stable-diffusion file, the model they asked me to get + deleting a local copy of anaconda for one specifically in the program data folder... only to find out that i had one in both program data + in users. how lucky.

Anyways, i managed to get a problem where Remote host? keeps forcibly closing the link severing the download, how do i fix this?
what am i doing wrong? i can't find anything anywhere on how the fuck to make this stop and I'm going mad as all Q&A + errors do not include anything related to my own issues. 

("Connection broken: ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)", ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))

EDIT: i made an account just so i could cry about this on an online forum

**Comment by Sharrnah** [comment]
Nothing to cry about.
I recommend using Docker and WSL2. that way you don't have to fight with local installation of all the tools and it dependencies.

If you still find the official Dockerimage still too complicated, i am trying to make it even easier in my repo here:
https://github.com/Sharrnah/Stable-Diffusion-Docker

where i even have uploaded the docker image to docker hub. (but i won't update it every time something changes.
Only when i find the current official repo is stable enough for my taste)

official and my docker project also download the AI models automatically when not found.

---------

specially for your issue:
I had a friend with a similar issue i think. In the end i think it was a Firewall issue that broke the connections.

**Comment by Kiekendeko** [comment]
if it was the firewall how would i even fix that, what would be the thing im looking to allow through?
its the easiest part of the hypothetical fixes, and im not particularly looking foward to using Docker and WSL2.

**Reply by Sharrnah** [reply]
Just tried to help here and give a suggestion to maybe have a simpler installation.

what exactly does not download fully?
Is it some AI Model file? Tried to download it manually?

**Reply by Kiekendeko** [reply]
its only the parts in the CMD download sections and only those, anything i manually download turns out fine but CMD downloading usually doesnt work if its over a certain size, i failed to download the pytorch file for a long ass time, and when i chose to switch to https://github.com/cmdr2/stable-diffusion-ui/tree/v2#installation this particular version refused to finish downloading the last file required to run the software. : / im kinda defeated.
